test/functional/lua/highlight_spec.lua

```diff
@@ -126,9 +126,10 @@ describe('vim.highlight.on_yank', function()
       vim.api.nvim_buf_set_mark(0,"]",1,1,{})
       vim.highlight.on_yank({timeout = math.huge, on_macro = true, event = {operator = "y"}})
     ]])
-    neq({}, api.nvim__win_get_ns(0))
+    local ns = api.nvim_create_namespace('hlyank')
+    eq(api.nvim_get_current_win(), api.nvim__ns_get(ns).wins[1])
     command('wincmd w')
-    eq({}, api.nvim__win_get_ns(0))
+    neq(api.nvim_get_current_win(), api.nvim__ns_get(ns).wins[1])
```

Comment:
can we turn this into an eq() test somehow (same for the other case below). that is a much strong test.

```lua
    local win = api.nvim_get_current_win()
    eq(win, api.nvim__ns_get(ns).wins[1])
    command('wincmd w')
    eq(win, api.nvim__ns_get(ns).wins[1])
```